2-mandatory/2-financial-times.js

```diff
@@ -14,25 +21,51 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+   const separateTitleArray = [];
```

Comment:
Have a think if you really need to create the `separateTitleArray`. Maybe you can use reduce on the `allArticleTitles` directly?